src/compile/scale/parse.ts

```diff
@@ -47,25 +48,37 @@ export function parseScale(model: UnitModel, channel: ScaleChannel) {
   const scale = model.scale(channel);
   const sort = model.sort(channel);
 
-  const scaleComponent: VgScale = {
-    name: model.scaleName(channel + '', true),
-    type: scale.type,
-    domain: parseDomain(model, channel),
-    range: parseRange(scale)
-  };
+  // FIXME parseScale should return something similar to Split<VgScale>
```

Comment:
Good catch.  This is actually done  in the last two commits of this PR.  I'll remove the comment.